modules/services/buildkite-agent.nix

```diff
@@ -17,7 +17,7 @@ let
 
   hooksDir = let
     mkHookEntry = name: value: ''
-      cat > $out/${name} <<EOF
+      cat > $out/${name} <<'EOF'
```

Comment:
Perhaps this should use `writeScript` to avoid other edge-cases like eg. `EOF`.

Reply:
OK good point -- I have added a commit which fixes that.